conf/app.routes

```diff
@@ -1,6 +1,6 @@
 # microservice specific routes
 
-GET   /resources  @controllers.ResourceLinksController.get
+GET     /resources                                  controllers.ResourceLinksController.get
 
 POST /reference-data-lists                          controllers.ReferenceDataController.referenceDataLists()
-POST /customs-office-lists/customs-office-lists     controllers.ReferenceDataController.customsOfficeLists()
```

Comment:
We can't change this as yet, this is what's in the spec, `/customs-office-lists/customs-office-lists`. We need to try to get this changed but for now that's what we need to implement

Reply:
Ah sorry, will revert this